website/frontend/src/components/blocks/MessagesList/MessagesList.tsx

```diff
@@ -47,10 +48,12 @@ export const MessagesList: VFC<Props> = ({ messages }) => {
               <p className="messages__list-caption">{message.destination && fileNameHandler(message.destination)}</p>
             </div>
             <div className="messages__list-item">
-              <p className="messages__list-link">{message.id}</p>
+              <Link className="messages__list-link" to={`/message/${message.id}`}>
+                <p className="messages__list-text">{message.id}</p>
+              </Link>
               <div className="programsCopyId">
                 <button type="button" onClick={() => copyToClipboard(message.id, alert, 'Message ID copied')}>
-                  <img src={copyIcon} alt="copy program ID" />
+                  <img src={copyIcon} alt="copy message ID" />
```

Comment:
Why we didn't use icons icons set?

Reply:
The icons from the set is not similar to our design